src/libstd/rt.rs

```diff
@@ -65,3 +66,10 @@ fn lang_start(main: *const u8, argc: isize, argv: *const *const u8) -> isize {
         0
     }
 }
+
+/// Fixed frame used to clean the backtrace with `RUST_BACKTRACE=1`.
+#[no_mangle]
+#[inline(never)]
+pub fn __rust_begin_backtrace_binary(f: fn()) {
```

Comment:
Ignore.

I've just realized though, since we use `contains` and not exact matching, then `no_mangle` can be just removed most probably.